src/briefcase/platforms/linux/system.py

```diff
@@ -994,6 +1023,104 @@ def _package_rpm(self, app: AppConfig, **kwargs):
             self.distribution_path(app),
         )
 
+    def _package_pkg(self, app: AppConfig, **kwargs):
+        self.logger.info("Building .pkg.tar.zst package...", prefix=app.app_name)
+
+        # The description *must* exist.
+        # pkgdesc has 80 char limit.
+        if app.description is None:
+            raise BriefcaseCommandError(
+                "App configuration does not define `description`. "
+                "Arch projects require a description."
+            )
+        # The changelog should exist.
+        changelog_source = self.base_path / "CHANGELOG"
+        if not changelog_source.exists():
```

Comment:
is `if not ((changelog_source.is_file()) and (changelog_source.exists())) good

Reply:
It works but it's overkill; `exists()` simply verifies that `stat` returns something for the path while `is_file()` confirms that `stat` believes the path is a regular file (or symlink). So, `is_file()` effectively implies `exists()` making the `exists()` call redundant.

Reply:
ok should I also use `is_file()` in the tests also?

Reply:
ehh, it's less important, per se, in the tests since we already know it's a regular file....because the test just made the file right before the test ran. The use of `is_file()` here is just to protect against unexpected user behavior where the changelog is a directory or something weird.